extensions/hatena-bookmark/src/useStories.tsx

```diff
@@ -0,0 +1,86 @@
+import os from "node:os";
+import { useEffect, useState } from "react";
+import { Cache, environment } from "@raycast/api";
+import Parser from "rss-parser";
+
+const cache = new Cache();
+const CACHE_DURATION = 300000;
+type CacheEntry = { timestamp: number; items: Parser.Item[] };
+
+export enum Topic {
+  総合 = "hotentry.rss",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

Reply:
@pernielsentikaer 
Thank you for your comment!
Is it OK if I change this list from Japanese to English?

Reply:
I fixed it your review commet! Is this fix OK?